t/api/10-jobgroups.t

```diff
@@ -95,15 +96,37 @@ subtest 'list job groups' => sub() {
 };
 
 subtest 'create parent group' => sub() {
+    for my $form (
+        {size_limit_gb                  => '-300GB'},
+        {build_version_sort             => 'true'},
+        {keep_logs_in_days              => '18 days'},
+        {keep_important_logs_in_days    => '4 days'},
+        {keep_results_in_days           => '30 days'},
+        {keep_important_results_in_days => '1 days'},
+        {default_priority               => 'inherit'},
+        {carry_over_bugrefs             => 'no'},
+      )
+    {
```

Comment:
Not critical, but i'm not really a fan of this formatting. I'd probably declare the data structure as an array before the `for` loop.